### DISCUSSION TITLE
Ti.App.getArguments() missing in SDK 10.x

### DISCUSSION THREAD

**Discussion by caspahouzer** [appcelerator/titanium_mobile#12801] [Q&A] [answered]
I tried SDK 10 to get use of `OptionBars`, mentionend in https://github.com/appcelerator/titanium_mobile/discussions/12790

Now I'm missing https://titaniumsdk.com/api/titanium/app.html#getarguments

It's not deprecated as I can see in the docs. It's also not in the `api.jsca`

**Comment by ewanharris** [comment, marked as answer]
@caspahouzer, does `Ti.App.arguments` work ok for you? Personally, I think this might be on where we should view the docs as invalid instead of the runtime as (refactoring cost aside) there is not much value in maintaining a getter for something that functions ok as a property

**Reply by caspahouzer** [reply]
Yes, that did the trick. Thanks Ewan
